### PR TITLE
Update introducing-metrics-in-innersource.md

### PR DESCRIPTION
**Pull request by arlou**
I consolidated some metrics candidates into the pattern from forums and InnerSource Commons slack channel. Hope that helps.

### CODE REVIEW

File: patterns/1-initial/introducing-metrics-in-innersource.md [hunk 1]
```diff
@@ -89,8 +100,11 @@ Initial
...
 - Russ Rutledge
 - Tom
+- Jack Yang
 
 ## Acknowledgement
 
 - Georg
 - Bob
+- Aaron Steward
+- Willson Mar
```
Comment (spier) [inline patterns/1-initial/introducing-metrics-in-innersource.md]:
```suggestion
- Wilson Mar
```

Possibly spelled like this?

Also did you check with both Aaron and Wilson that they would like to be mentioned here?

File: patterns/1-initial/introducing-metrics-in-innersource.md [hunk 2]
```diff
@@ -64,7 +64,18 @@ Produce a characterization of metrics as this might be helpful for others to und
 * Another class of metrics is around how much InnerSourcing is happening. A raw definition of InnerSource based on code submissions to a repository not owned by the submitter's team could be measurable in a few ways.
 * A third class of metrics are KPIs that we believe will improve the raw amount of InnerSourcing happening (e.g. mean-time-to-review, automated test runs on PRs, etc.).
 
-Note: The aforementioned metrics are some common examples. They aren't a one-size-fits-all. We expect the business goals of the organization to drive the metric selection process.
+Note: The aforementioned metrics are some common examples. They aren't a one-size-fits-all. We expect the business goals of the organization to drive the metric selection process. Below is the list consolidating some metrics candidates by business goals.
```
Comment (spier) [inline patterns/1-initial/introducing-metrics-in-innersource.md]:
```suggestion
_Note:_ The aforementioned metrics are some common examples. They aren't a one-size-fits-all. We expect the business goals of the organization to drive the metric selection process. 

The list below consolidates some metrics candidates, grouped by business goals.

```

File: patterns/1-initial/introducing-metrics-in-innersource.md [hunk 3]
```diff
@@ -64,7 +64,18 @@ Produce a characterization of metrics as this might be helpful for others to und
...
+|---|---|
+| Remove silos and bottlenecks | # of PRs from external contributors<br># of PRs from external contributors that are merged<br># of outside collaborators<br># of issues created from external contributors<br># of reviews from external contributors |
+| Knowledge sharing | # of clone or forks from external guests<br># of visits from external guests (proposed by Jack) |
+| Creating reusable software | # of submodule usage<br># of code reuse across projects<br># of clone or forks from external guests |
+| Increase development speed | Time to market<br>Mean-time-to-review |
+| Employee satisfication& talent retention |  |
+| Improve quality | # of automated test runs on PRs<br># of issues created from external contributors |
+| Innovation |  |
+| Others | # of repositories using InnerSource components (managed contributing guidelines, assigned TCs)<br>Days of host's responsiveness on the issues/PRs created by external contributors |
+
```
Comment (spier) [inline patterns/1-initial/introducing-metrics-in-innersource.md]:
```suggestion

```

Fix linter issue

File: patterns/1-initial/introducing-metrics-in-innersource.md [hunk 4]
```diff
@@ -64,7 +64,18 @@ Produce a characterization of metrics as this might be helpful for others to und
...
 * A third class of metrics are KPIs that we believe will improve the raw amount of InnerSourcing happening (e.g. mean-time-to-review, automated test runs on PRs, etc.).
 
-Note: The aforementioned metrics are some common examples. They aren't a one-size-fits-all. We expect the business goals of the organization to drive the metric selection process.
+Note: The aforementioned metrics are some common examples. They aren't a one-size-fits-all. We expect the business goals of the organization to drive the metric selection process. Below is the list consolidating some metrics candidates by business goals.
+| Business Goals | Metric Candidates |
+|---|---|
+| Remove silos and bottlenecks | # of PRs from external contributors<br># of PRs from external contributors that are merged<br># of outside collaborators<br># of issues created from external contributors<br># of reviews from external contributors |
+| Knowledge sharing | # of clone or forks from external guests<br># of visits from external guests (proposed by Jack) |
```
Comment (spier) [inline patterns/1-initial/introducing-metrics-in-innersource.md]: Do you want to keep this "(proposed by Jack)" note in here?

We don't have to mention where which metric comes from. It is actually quite likely that many organizations have discover the same metrics over and over again anyways :)

**Review by spier — CHANGES_REQUESTED** [review]
Great idea to add metric candidates @arlou! Thanks for this.

What would you think about adding the links to the public forums/websites where you found some of this information, in case people want to read more about this. We cannot reference the conversations in the ISC Slack (as they are under Chatham House Rule) but we can reference anything that's already public.

**Comment by spier** [comment]
@dicortazar you were quite involved with this patten, weren't you?

Could you review the metric candidates proposed by @arlou?

Also are there any Known Instances already of this pattern being used in an org?
This would allow us to level up the pattern even further.

File: patterns/1-initial/introducing-metrics-in-innersource.md [hunk 5]
```diff
@@ -64,8 +64,17 @@ Produce a characterization of metrics as this might be helpful for others to und
...
+| Creating reusable software | # of submodule usage<br># of code reuse across projects<br># of clone or forks from external guests |
+| Increase development speed | Time to market<br>Mean-time-to-review |
+| Employee satisfication& talent retention |  |
+| Improve quality | # of automated test runs on PRs<br># of issues created from external contributors |
+| Innovation |  |
+| Others | # of repositories using InnerSource components (managed contributing guidelines, assigned TCs)<br>Days of host's responsiveness on the issues/PRs created by external contributors |
 ## Resulting Context
```
Comment (spier) [inline patterns/1-initial/introducing-metrics-in-innersource.md:82]:
```suggestion

## Resulting Context
```

File: patterns/1-initial/introducing-metrics-in-innersource.md (hunk 4 above)
Comment (arlou) [reply patterns/1-initial/introducing-metrics-in-innersource.md]: Sorry, that's typo. I should remove it from the sentence

Comment (arlou) [reply patterns/1-initial/introducing-metrics-in-innersource.md]: BTW, I will contact Wilson and Aaron to check if they concern on being mentioned in the page; Thanks for @spier 's review and reminding

Comment (arlou) [reply patterns/1-initial/introducing-metrics-in-innersource.md]: Confirmed no concern from Aaron, and waiting for Wilson's confirmation

**Comment by spier** [comment]
Note to self for final cleanup:

* switch Problem and Context (to match the pattern template)
* turn Context and Forces into bullets (kind of like a checklist)

**Comment by spier** [comment]
@arlou I noticed three terms in the table that you added that seem similar but maybe not quite the same:

- external contributor
- outside collaborator
- external guest

Are those all have different meanings?

**Comment by arlou** [comment]
> @arlou I noticed three terms in the table that you added that seem similar but maybe not quite the same:
> 
> * external contributor
> * outside collaborator
> * external guest
> 
> Are those all have different meanings?

Hi @spier , thanks for checking. I think "external contributor" and "outside collaborator" are the same to describe the person who contributes to the shared repo from outsides. For consistency, I'd like to use "external contributor" for the role in this pattern description. For external guests, I think it is a little bit different from the rest, and it targes the person who just visit the shared repo to consume the content without contribution. My 2 cents

File: patterns/1-initial/introducing-metrics-in-innersource.md (hunk 4 above)
Comment (arlou) [reply patterns/1-initial/introducing-metrics-in-innersource.md]: Update, just got reply from Wilson. Confirmed he has no concern on this.

File: patterns/1-initial/introducing-metrics-in-innersource.md [hunk 6]
```diff
@@ -64,7 +64,20 @@ Produce a characterization of metrics as this might be helpful for others to und
...
+
+The list below consolidates some metrics candidates, grouped by business goals.
+
+| Business Goals | Metric Candidates |
+|---|---|
+| Remove silos and bottlenecks | # of PRs from external contributors<br># of PRs from external contributors that are merged<br># of external contributors<br># of issues created from external contributors<br># of reviews from external contributors |
+| Knowledge sharing | # of clone or forks from external guests<br># of visits from external guests |
+| Creating reusable software | # of submodule usage<br># of code reuse across projects<br># of clone or forks from external guests |
+| Increase development speed | Time to market<br>Mean-time-to-review |
+| Employee satisfication& talent retention |  |
```
Comment (spier) [inline patterns/1-initial/introducing-metrics-in-innersource.md]:
```suggestion
| Employee satisfaction & talent retention |  |
```

Fix typo and whitespace

File: patterns/1-initial/introducing-metrics-in-innersource.md [hunk 7]
```diff
@@ -89,8 +102,11 @@ Initial
...
 - Russ Rutledge
 - Tom
+- Jack Yang
 
 ## Acknowledgement
 
 - Georg
 - Bob
+- [Aaron Stewart](https://github.com/a-a-ron/innersource-template-pluralsight/tree/master/metrics)
+- Wilson Mar
```
Comment (spier) [inline patterns/1-initial/introducing-metrics-in-innersource.md:112]: @arlou do you have any online resource by Wilson Mar that you want to reference here?

**Comment by spier** [comment]
@dicortazar @NewMexicoKid @rrrutledge the metrics candidates contributed in this PR seem like a great extension of the pattern to me, as they give people possible starting points to figure out what the right things to measure might be for them in their org.

This pattern is still in the Initial stage (as we don't have a confirmed Known Instance yet).
Therefore I would be happy to merge this PR as is, and improve the pattern further in subsequent PRs.

Will wait for feedback until Thursday, so that we don't block this PR for much longer.

File: patterns/1-initial/introducing-metrics-in-innersource.md (hunk 7 above)
Comment (arlou) [reply patterns/1-initial/introducing-metrics-in-innersource.md:112]: hi @spier, thanks for checking; So far, no referred link on Wilson as I referred to his inputs from Slack channel.

**Review by spier — APPROVED** [review]
LGTM

**Pull request merged**